test/decl/ext/extensions_no_type.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: %target-swift-ide-test -print-indexed-symbols -source-filename %s
+
+struct Foo { }
+
+//===--- Test that we don't crash when validating members inside an extension with no type name.
+extension { // expected-error {{expected type name in extension declaration}}
+  static func ==(lhs: Foo, rhs: Foo) -> Bool {} // expected-error {{operator '==' declared in extension must be 'static'}}
+}
```

Comment:
I think we should move the test to `test/Index` now that we use this command.
Also, since the test is somewhat unusual, can you name it after the SR to provide more context? This will make it easier to move back to `decl/ext` once we fix the validation issue.

Reply:
Oh no, the crash is guarded, but the `expected-error` checks here will have no effect. The only way to test that your error is actually there is to `CHECK:` for it in the command output. That's a pretty dirty workaround, so let's wait for what Pavel has to say.

Reply:
I renamed the the test file to SR-12460.swift.

I don't understand your comment on <CHECK:>, the test seems to work correctly with those compile options? What I am missing?

Reply:
> I don't understand your comment on CHECK:, the test seems to work correctly with those compile options? What I am missing?

The test will still pass if you mess up the expected errors. You need `%target-typecheck-verify-swift` for the `expected-error` check to take effect, but the error you want is only emitted as a byproduct of the `%target-swift-ide-test -print-indexed-symbols` output.

Reply:
I don’t think typecheck-verify command is really necessary here, this output could be verified with FileCheck. I’ll make some experiments tomorrow and let you know.